README.md

```diff
@@ -82,6 +82,7 @@ raccoon.recommendFor('chrisId', 10, function(results){
 raccoon.config.nearestNeighbors = 5;  // number of neighbors you want to compare a user against
 raccoon.config.className = 'movie';  // prefix for your items (used for redis)
 raccoon.config.numOfRecsStore = 30;  // number of recommendations to store per user
+raccoon.config.updateRatingOnInput = true;  // update recommendations on each input
```

Comment:
Yes - take a look at lib/input.js line 15.

it will rating update will trigger on each call If you not put it in config